components/components.css

```diff
@@ -9,10 +9,12 @@
 @import url("../components/inputs/inputs.css");
 @import url("../components/reponsive-images/responsive-images.css");
 @import url("../components/cards/cards.css");
+@import url("../components/ratings/ratings.css");
 body{
     font-family: 'Roboto', sans-serif;
 }
 :root{
+    --primary-color:#4d4dff;
     /* Alerts */
     /* Blue */
```

Comment:
I didnt get what is this comments for?

Reply:
Actually I wrote it for my understanding but then forgot to remove them before committing . 
Thanks for pointing that out , I'll remove them